hosts_example.yml

```diff
@@ -65,57 +65,58 @@ all:
 
 zookeeper:
   hosts:
-    ip-172-31-34-246.us-east-2.compute.internal:
+    ip-172-31-56-107.ec2.internal:
```

Comment:
can we keep the old host names in this file? 
BC my commented out keytab examples match the old hostnames

Reply:
Corrected.